components/Header/Header.tsx

```diff
@@ -43,12 +34,17 @@ const Header = () => {
       <Logo />
 
       <div className="mx-5 flex items-center space-x-3 text-white lg:inline-flex">
-        <HoverMenuOption options={['Lerneinheit suchen']}>
+        <HoverMenuOption options={[{ name: 'Lerneinheit suchen', link: '/' }]}>
           <AcademicCapIcon className="header-icon" />
           <p className="text">Lerneinheit</p>
         </HoverMenuOption>
 
-        <HoverMenuOption options={['Cluster suchen', 'Cluster erstellen']}>
+        <HoverMenuOption
+          options={[
+            { name: 'Cluster suchen', link: '/' },
+            { name: 'Cluster erstellen', link: '/cluster/create' },
```

Comment:
der link zu cluster create wär eigentlich wieder ein eigener task gwesen..schreib da halt dann rein, wo du das mitgmacht hast

Reply:
Ich wollte da einen zweiten Commit machen, hab aber anscheinend davon vergessen und stattdessen ein amend Commit gemacht. Ich werde diese Änderung zurücknehmen und ein Commit in einem anderen Branch machen.